### ISSUE TITLE
Very strange issues with Checkerboard and Argyle patterns in images

### ISSUE THREAD

**Issue by UiharuKazari2008** [KichangKim/DeepDanbooru#83]
I have notices a very stange trend when processing images with images that contain the following
* Checkerboard Patterns
* Argyle Pattern
* Stars and Blue Night Sky's

Repeating patterns in general seem to really throw it off bad for some reason is this something that can be fixed in a better model or should i do some sort of tag cleaning.

Example:
It def has issues with "3d" art the most
<img width="960" alt="Screen Shot 2023-01-07 at 21 28 24" src="https://user-images.githubusercontent.com/15165770/211178003-7eda7490-a6cd-4606-97e1-68b5cfe22a9d.png">
but still applies with anime art
<img width="960" alt="Screen Shot 2023-01-07 at 21 36 48" src="https://user-images.githubusercontent.com/15165770/211178089-20c529b9-caf0-46f2-9250-de47410986e5.png">
The image in the results has about 244 tags 
<img width="756" alt="Screen Shot 2023-01-07 at 21 38 09" src="https://user-images.githubusercontent.com/15165770/211178125-c87d8645-5a28-4c77-961b-c351c76a64db.png">
Most of them being pretty much being pattern and all having very high rating too.(It cuts off because it overflowed the col in SQL)
![image](https://user-images.githubusercontent.com/15165770/211178123-09fdf177-996f-4228-acaa-56b9f0d85137.png)

I am working on trying to find a commonality in the tags to figure out how to mute them or group them all as "pattern"

**Comment by KichangKim** [comment]
Yes, checkerboard pattern is one of weak point of DeepDanbooru (#54). It is future work of DeepDanbooru, but I have no plan yet.
